geode-redis/src/main/java/org/apache/geode/redis/internal/RedisStats.java

```diff
@@ -206,6 +294,7 @@ public void endExpiration(long start) {
       stats.incLong(expirationTimeId, getTime() - start);
     }
     stats.incLong(expirationsId, 1);
+    expirations.incrementAndGet();
```

Comment:
I don't think we are counting this statistic properly.  According to the Redis `INFO` docs (and also testing our version vs. Redis), it should be `the number of keys with an expiration` since we are using it in the `keyspace` section:
```
The keyspace section provides statistics on the main dictionary of each database. The statistics are the number of keys, and the number of keys with an expiration.

For each database, the following line is added:

dbXXX: keys=XXX,expires=XXX
```

This means that we should add a key when it is set to expire and remove it when it no longer has an expiration (because it was persisted, expired on its own, was deleted, etc.)

Reply:
story added to backlog for this

Reply:
story made for this